command/secrets_tune.go

```diff
@@ -158,6 +159,14 @@ func (c *SecretsTuneCommand) Flags() *FlagSets {
 			"the plugin catalog, and will not start running until the plugin is reloaded.",
 	})
 
+	f.StringSliceVar(&StringSliceVar{
+		Name:   flagNameDelegatedAuthAccessors,
+		Target: &c.flagDelegatedAuthAccessors,
+		Usage: "A list of permitted authentication accessors this backend can delegate authentication too. " +
```

Comment:
s/too/to

Reply:
Fixed in https://github.com/hashicorp/vault/pull/23924/commits/65f309c7382a5ae2016558d12f4705a85bca1b85. Also added another failure case of referencing an auth mount that doesn't exist but had been 'allowed'